x-pack/plugins/apm/public/components/app/Correlations/index.tsx

```diff
@@ -19,10 +19,10 @@ import {
 } from '@elastic/eui';
 import { useHistory } from 'react-router-dom';
 import { enableCorrelations } from '../../../../common/ui_settings_keys';
-import { useApmPluginContext } from '../../../hooks/useApmPluginContext';
+import { useApmPluginContext } from '../../../context/apm_plugin_context/use_apm_plugin_context';
 import { LatencyCorrelations } from './LatencyCorrelations';
 import { ErrorCorrelations } from './ErrorCorrelations';
-import { useUrlParams } from '../../../hooks/useUrlParams';
+import { useUrlParams } from '../../../context/url_params_context/use_url_params';
```

Comment:
Maybe to keep consistency we could call it `useUrlParamsContext`, along side to all hooks that are inside the `context` folder.

Reply:
sgtm 👍